bom/pom.xml

```diff
@@ -102,6 +102,16 @@
         <version>1.2.1</version>
         <optional>true</optional>
       </dependency>
+      <dependency>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-annotations</artifactId>
+        <version>3.1.12</version>
+      </dependency>
+      <dependency>
+        <groupId>com.github.stephenc.jcip</groupId>
+        <artifactId>jcip-annotations</artifactId>
+        <version>1.0-1</version>
+      </dependency>
```

Comment:
Required to add these now since these dependencies were moved to provided scope as part of [CASSJAVA-34](https://issues.apache.org/jira/browse/CASSJAVA-34) (see [this commit](https://github.com/apache/cassandra-java-driver/commit/1991537d50b9100a3d25cd669af2a892409c1c95))